### PR TITLE
Draft posts and year in review

### PR DESCRIPTION
**Pull request by jnfoster**
This pull request does two things:

1. It adds support for draft blog posts that are published in at [p4.org/blog/drafts](https://p4.org/blog/drafts).

2. It adds a first draft for a year-in-review blog post.

### CODE REVIEW

**Comment by jnfoster** [comment]
I'm going to merge this right away so I can get reviews from others. Nothing is visible to the public unless you navigate to the drafts page.

File: _drafts/year-in-review.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## New Version of the P4 Language
+
+In May, the [Language Design Working
+Group](https://p4.org/working-groups/) released [version
+1.0.0](https://p4.org/p4-spec/docs/P4-16-v1.0.0-spec.html) of the
+P4~16 language. P4~16 retains the core features of the original
+version of the language, now called P4~14, but adds new primitives
+designed to make it easy to write safe, portable, and modular data
+plane programs. See the [blog
+post[(https://p4.org/technical-steering-committee/get-involved-with-shaping-p4s-future.html)
```
Comment (cc10512) [inline _drafts/year-in-review.md]: link formatting seems wrong. Should be: `See our [P4<sub>16</sub> announcement](https://p4.org/technical-steering-committee/get-involved-with-shaping-p4s-future.html) for more details.`

File: _drafts/year-in-review.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+portable "extern" primitives as well as a standard packet-processing
+pipeline that can be mapped to different underlying hardware and
+software targets.
+
+## Flexible Control-Plane API
+
+To date, much of the focus in the P4 community has been on designing
+constructs for data plane programming. But in any practical
+deployment, P4 programmers will also need to write control-plane
+software. In November, the [API Working
```
Comment (cc10512) [inline _drafts/year-in-review.md]: Doesn't parse in current form. In November, the API Working group announced  the initial implementation ...

File: _drafts/year-in-review.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+provides a new way for control plane software to manage a huge variety
+of different programmable and fixed-function switches. See the [blog
+post](https://p4.org/api/p4-runtime-putting-the-control-plane-in-charge-of-the-forwarding-plane.html)
+on P4 Runtime for more details.
+
+## Emerging Applications
+
+As P4 continues to mature, there is growing interest in applications
+that change the way people design, build, and run their networks. In
+November, to help nurture an ecosystem that is well aligned with the
```
Comment (cc10512) [inline _drafts/year-in-review.md]: I would drop In November, and start directly with To help nurture.

File: _drafts/year-in-review.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+open-source software for novel data plane applications enabled by
+P4. Initially, this working group will focus on data plane telemetry
+but it may broaden its scope in the future as new applications
+emerge. See the [blog
+post](https://p4.org/members/announcing-the-p4-applications-working-group.html)
+announcing the Applications Working Group for more details.
+
+## Community Events
+
+We organized a number of community events including:
```
Comment (cc10512) [inline _drafts/year-in-review.md]: Our growing community has come together several times during the year at the following events:

**Comment by jnfoster** [comment]
Thanks. I've created a new PR with these fixes: https://github.com/p4lang/p4lang.github.io/pull/29

**Pull request merged**